recipes/libftdi/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.5" %}
+{% set sha256 = "7c7091e9c86196148bd41177b4590dccb1510bfe6cea5bf7407ff194482eb049" %}
+
+package:
+  name: libftdi
+  version: '{{ version }}'
+
+source:
+  fn:   libftdi1-{{ version }}.tar.bz2
+  url:  https://www.intra2net.com/en/developer/libftdi/download/libftdi1-{{ version }}.tar.bz2
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+    - pkgconfig
+  host:
+    - libusb
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/libftdi1/ftdi.h  # [unix]
+    - test -f ${PREFIX}/lib/libftdi1${SHLIB_EXT}  # [osx]
+    - test -f ${PREFIX}/lib/pkgconfig/libftdi1.pc  # [osx]
+    - test -f ${PREFIX}/lib/cmake/libftdi1/LibFTDI1Config.cmake  # [osx]
+    - test -f ${PREFIX}/lib64/libftdi1${SHLIB_EXT}  # [linux]
```

Comment:
We should  use "-DCMAKE_INSTALL_LIBDIR=lib" to install into lib, not lib64. 

https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html

Reply:
Hmm, I already pass this argument: https://github.com/conda-forge/staged-recipes/blob/be6929f34060f3aa06d9fbb6c9d9b81bf3debb5b/recipes/libftdi/build.sh#L11

Reply:
https://github.com/lipro/libftdi/blob/9da4394efeafa2fcfdc809a9bec3219c0a95e77a/CMakeLists.txt#L52-L61

Looks like we need `-DLIB_SUFFIX=""`